tests/repegAndSpread.ts

```diff
@@ -873,6 +876,8 @@ describe('repeg and spread amm', () => {
 				driftClientUserI.getSpotMarketAssetValue(),
 				QUOTE_PRECISION
 			);
+			await driftClientI.fetchAccounts();
+			await driftClientUserI.fetchAccounts();
```

Comment:
any idea why the previous fetchAccounts didnt work?

https://github.com/drift-labs/protocol-v2/pull/582/files#diff-f8a7084885668ddd2e9627a28a4c8817fbe46aa82fa5cfb2b5b7c070fe23abceR870

Reply:
oh i dont think you fetched for the `driftClientUserI`